mod-users/src/test/java/org/folio/ModUsersTests.java

```diff
@@ -0,0 +1,32 @@
+package org.folio;
+
+import org.folio.test.TestBase;
+import org.folio.test.config.TestModuleConfiguration;
+import org.folio.test.services.TestIntegrationService;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class ModUsersTests extends TestBase {
+
+  private static final String TEST_BASE_PATH = "classpath:domain/mod-users/features/";
+
+  public ModUsersTests() {
+    super(new TestIntegrationService(new TestModuleConfiguration(TEST_BASE_PATH)));
+  }
+
+  @BeforeAll
+  public void setup() {
+    runFeature("classpath:domain/mod-users/users-junit.feature");
+  }
+
+  @AfterAll
+  public void tearDown() {
+    runFeature("classpath:common/destroy-data.feature");
+  }
+
+  @Test
+  void loanTest() {
```

Comment:
Why is this called `loanTest` when it is a set of tests about users?

Reply:
Yes i miss this , i will correct it .